Spanner/src/Database.php

```diff
@@ -1691,6 +1691,16 @@ public function createSession(array $options = [])
         return $this->operation->createSession($this->name, $options);
     }
 
+    /**
+     * @param int $count
+     * @param array $options
+     * @return Session[]
+     */
+    public function createSessionsAsync($count, array $options = [])
```

Comment:
Similar to `deleteSessionAsync` let's not expose these so clients can interact with directly interact with them just yet. (Applies to the `Operation` class as well)

Reply:
👍